src/test/java/io/confluent/examples/streams/window/CustomWindowTest.java

```diff
@@ -61,220 +61,243 @@
 
     @Test
     public void shouldSumNumbersOnSameDay() {
-        final List<MyEvent> inputValues = Arrays.asList(
```

Comment:
Replacing `MyEvent` with new KIP-470 `TestRecord:
 - requires to pass in `null` key
 - required to pass timestamp as `Instant`